build.gradle

```diff
@@ -87,6 +89,8 @@ dependencies {
     implementation 'android.arch.lifecycle:extensions:1.1.1'
     implementation 'android.arch.lifecycle:common-java8:1.1.1'
 
+    implementation 'com.google.firebase:firebase-messaging:18.0.0'
```

Comment:
Let's define the version at the top of the file, like we did for the other dependencies we added